### DISCUSSION TITLE
记一次Github提交PR过程

### DISCUSSION THREAD

**Discussion by kuizuo** [kuizuo/blog#147] [General]
https://kuizuo.cn/reference/记一次Github提交PR过程
故事起因

**Comment by sunzehui** [comment]
这英语尬的我用脚指头抠出来了个三室一厅

**Comment by kuizuo** [comment]
> @sunzehui
> 这英语尬的我用脚指头抠出来了个三室一厅

高中英语就没及格过,甚至大学英语都挂科过,不过丝毫不影响编写代码

**Comment by hanchengxu** [comment]
分享的经历很值得我们学习，至少值得我学习👍。
